docs/digger/admin/managing-digger-logs-artifacts.asciidoc

```diff
@@ -0,0 +1,31 @@
+== Managing Digger Build Logs and Artifacts
+App builds started by Digger produce build logs and build artifacts.
+Over time, these will build up. There are a number of options to manage the buildup. If you are using Digger with RHMAP, these options can be specified
+by setting environment variables in the `millicore` deployment(???) configuration in
```

Comment:
q0: Does this (managing digger build logs and artifacts) apply to upstream as well? (e.g. names of the environment variables are the same) Can the variables be set in upstream as well? Or should we include this document only in downstream repo? 
q1: Environment variable vs RHMAP property. Is it true that these two terms point to the same thing? If yes, i suggest to keep the terminology consistent and simple - use Environment variable.

Playbook is 404 btw.